v1/related.js

```diff
@@ -45,7 +46,7 @@ class Related {
             delete res.body.items;
 
             // Step 2: Hydrate response as always.
-            return mwUtil.hydrateResponse(res, (uri) => mwUtil.fetchSummary(hyper, uri));
+            return mwUtil.hydrateResponse(res, (uri) => mwUtil.fetchSummary(hyper, uri, rh));
```

Comment:
You probably would want to add `vary` and `content-language` here. Conditionally on `mwUtil.shouldConvertAcceptLanguage` or based on one of the summary responses.

Reply:
Also, just to make sure, clone rh and delete 'cache-control' header.

Reply:
Done.